docs/source/index.rst

```diff
@@ -34,7 +34,7 @@ over 7,500 packages at repo.anaconda.com that are built, reviewed,
 and maintained by Anaconda\ |reg|.
 
 Conda can be combined with continuous integration systems such
-as Travis CI and AppVeyor to provide frequent, automated testing
+as GitHub Actions to provide frequent, automated testing
```

Comment:
I'd recommend setting "such as GitHub Actions" off in commas.

"Conda can be combined with continuous integration systems, such as GitHub Actions, to provide..."